src/test/java/games/strategy/test/EqualsPredicateTests.java

```diff
@@ -168,7 +174,8 @@ public void shouldReturnFalseWhenOwnerObjectsAreNotEqual() {
       }
     }
 
-    private static final class FakeOwneeClass {
+    @Nested
+    private final class FakeOwneeClass {
```

Comment:
This class does not contain tests, and thus does not need to be annotated with `@Nested`.